reliability-scanner/plugin/reliability-scanner-custom-values.lib.yml

```diff
@@ -0,0 +1,18 @@
+#@ def config():
+checks:
+- name: "Pod QOS check"
+  description: Checks each pod to see if the minimum desired QOS is defined.
+  kind: v1alpha1/pod/qos
```

Comment:
Is this `kind` used by only the configuration `checks:`?
Also, what tool generate the objects? Do you manually generate them?
Using do you foresee these objects being versioned ?  
Keeping track of different versions can be labor intensive.